wgengine/magicsock/magicsock.go

```diff
@@ -2468,8 +2386,68 @@ func (c *Conn) DERPs() int {
 	return len(c.activeDerp)
 }
 
-func (c *Conn) SetMark(value uint32) error { return nil }
-func (c *Conn) LastMark() uint32           { return 0 }
+// Bind returns the wireguard-go conn.Bind for c.
+func (c *Conn) Bind() conn.Bind {
+	return c.bind
+}
+
+// connBind is a wireguard-go conn.Bind for a Conn.
+type connBind struct {
+	*Conn
+	mu     sync.Mutex
+	closed bool
+}
+
+// Open is called by WireGuard to create a UDP binding.
+func (c *connBind) Open(uint16) ([]conn.ReceiveFunc, uint16, error) {
```

Comment:
I'd name the input parameter so it looks less mysterious. Maybe named it `portWeIgnoreFromWireguard` :)

also name result parameters, at least the port.